static/js/containers/DashboardPage.js

```diff
@@ -294,6 +297,17 @@ class DashboardPage extends React.Component {
     this.fetchCoupons()
     this.handleOrderStatus()
     this.fetchDiscussionsFrontpage()
+    this.fetchUserProfile()
+  }
+
+  fetchUserProfile = () => {
```

Comment:
we aren't actually fetching the user profile here (it's fetched by a lifecycle method in `App.js`) so I might suggest we call this instead `checkFinancialAidError` or something similar.